document/core/syntax/values.rst

```diff
@@ -145,6 +145,19 @@ Conventions
 
 * The meta variable :math:`z` ranges over floating-point values where clear from context.
 
+.. index:: ! simd, fixed-width, vector
+   pair: abstract syntax; packed simd number
+.. _syntax-simd:
+
+Fixed-Width SIMD
+~~~~~~~~~~~~~~~~
+
+Fixed-width SIMD data represents packed integers, or floating-point values. The packed values of a given SIMD value must all be of the same type, which can be any of the signed, unsigned integer types, and floating-point types ocurring in this proposal. The supported representations of the packed values are 4 32-bit, or 2 64-bit |IEEE754|_ numbers, or different widths of packed integer values specifically 2 64-bit integers, 4 32-bit integers, 8 16-bit integers, or 16 8-bit integers.
```

Comment:
Since SIMD values can be interpreted arbitrarily, the abstract syntax needs to pick one unique representation. Similar to integers, this then has to be reinterpreted by explicit meta-operators in contexts that determine a specific interpretation.